common/po/as.po

```diff
@@ -185,14 +183,14 @@ msgstr ""
 #: gutterball/src/main/java/org/candlepin/gutterball/report/StatusTrendReport.java:129
 msgid ""
 "Filter on subscriptions which have management enabled set to this value "
-"(boolean)"
+"(Boolean)"
```

Comment:
This change should coincide with the change in the code. If the lookup is case sensitive (map lookups are, typically), this string won't be found.